_posts/2023-02-22-user-defined-operators.md

```diff
@@ -0,0 +1,484 @@
+is the support for defining *operators* on [user defined value types](https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types) (UDVTs).
+
+If you have not been keeping up with recent features, UDVTs are a new class of types introduced in Solidity 0.8.8.
+They provide zero-cost abstraction of an elementary value type that results in a completely new type.
+This is similar to creating an alias, but the new type is distinct from the underlying value type
+and all other UDVTs derived from it.
+The ability to use operators is meant to bring the UDVTs closer to being as natural to use as the
+built-in types and promote the use of type-safe interfaces.
+Below, we are going to take a closer look into the details of this new mechanism.
+
+### A Quick Refresher
+Before we get into the details of the new feature and how it affects the language, it may be a good
+idea to first go over the two existing features that it builds upon: built-in operators and `using for`.
+
+#### Built-in operators
+Solidity, like the vast majority of programming languages,
+[offers a set of operators](https://docs.soliditylang.org/en/latest/types.html#operators) as syntactic sugar
+over common mathematical functions.
+The difference between operators and functions is purely syntactical, and the decision
+to choose one over the other is to a large extent arbitrary.
+What is included in that set, however, varies widely from language to language.
+
+Here is the full list for Solidity:
+
+| Category             | Operators                                                                |
+|----------------------|--------------------------------------------------------------------------|
+| Bitwise              | `&`, `\|`, `^`, `<<`, `>>`, `~`                                          |
+| Arithmetic           | `+`, `-`, `*`, `/`, `%`, `**`                                            |
+| Comparison           | `==`, `!=`, `<`, `<=`, `>`, `>=`                                         |
+| Boolean              | `!`, `&&`, `\|\|`                                                        |
+| Increment/decrement  | `++`, `--`                                                               |
+| Simple assignment    | `=`                                                                      |
+| Composite assignment | `+=`, `-=`, `^=`, `*=`, `/=`, `%=`, `&=`, `\|=`, `^=`, `<<=`, `>>=`      |
+| Miscellaneous        | `delete`, `new`, `.`, `()`, `[]`, `?:`                                   |
+
+Most of them require no explanation.
+In some cases, however, you may be surprised by what counts as an operator.
+The common infix operator, like `a + b`, that may be on your mind when you hear "operator" is
+not the only flavor operators come in.
+The last row of the table has items that to some users may not seem operator-like at all.
+`delete` and `new`  are in fact operators, and follow precedence and associativity rules despite not being symbols.
```

Comment:
```suggestion
`delete` and `new` are in fact operators and follow precedence and associativity rules despite not being symbols.
```

Reply:
Removed this paragraph due to this suggestion: https://github.com/ethereum/solidity-blog/pull/202#discussion_r1113390948